components/util/constants.tsx

```diff
@@ -120,3 +120,56 @@ export const AustralianStatesList = [
 		value: "100000005",
 	},
 ];
+
+export const Locations = [
+	{
+		label: "All",
+		value: "All",
+	},
+	{
+		label: "Sydney",
+		value: "Sydney",
+	},
+	{
+		label: "Brisbane",
+		value: "Brisbane",
+	},
+	{
+		label: "Melbourne",
+		value: "Melbourne",
+	},
+	{
+		label: "Newcastle",
+		value: "Newcastle",
+	},
+	{
+		label: "China",
+		value: "China",
+	},
+];
+
+export const Employment_Type = [
+	{
+		label: "Contract",
+		value: "Contract",
+	},
+	{
+		label: "Permanent",
+		value: "Permanent",
+	},
+	{
+		label: "Work Experience",
+		value: "Work Experience",
+	},
+];
+
+export const Job_Status = [
+	{
+		label: "Available",
+		value: "Available",
+	},
+	{
+		label: "Filled",
+		value: "Filled",
+	},
+];
```

Comment:
- Move to separate file in `/components/util/constants`
- Consider changing to array of strings